docs/blog.md

```diff
@@ -2,30 +2,30 @@
+The "average" web user is heavily reliant on "random access" style interaction using a mouse or touch screen. However, _all_ users can benefit from seeing where focus is when they're about to type into an input field. To lllustrate, picture using a login form in your mobile web browser. Here, the focus ring provides an indication of where your keystrokes are going to end up - critical information if you're about to type your password!
 
-Then, of course, there are users who use the keyboard heavily or exclusively. This may be
+Then, of course, there are users who use the keyboard heavily or exclusively. This may be:
 * A user with a disability precluding the fine motor control necessary to control a mouse;
 * A user whose mouse is broken;
 * A skilled user (you?) who finds it more efficient not to move their hands from the keyboard;
 
 ...or many other cases. 
  
-In any case, the `tab` key allows keyboard users to navigate sequentially through interactive elements on the page (such as form fields). The _only way_ for keyboard users to know which element is active (and thus how you might potentially interact with it via the keyboard) is via the focus style - i.e., by default, the __focus ring__).
+In any case, the `tab` key allows users to navigate sequentially through interactive elements on the page (such as form fields) with just a keyboard. In fact, the _only way_ for keyboard users to know which element is active (and thus how they might potentially interact with it via the keyboard) is via a visual indictor of what thing is focused - i.e., by default, the __focus ring__).
 
-Since keyboard users pretty much can't do anything without knowing where focus is, it's tempting to put our foot down and ignore the subtleties of the issue: we could simply say, "when a thing has focus it *must* get the focus ring indicator". Ok, since designs vary, what the particular focus ring should look like could vary too: on a white page background, a light blue outline might be really appropriate - but if your interface _is_ light blue, you might need to consider this in your branding and perhaps provide something like:
+Given this, it's tempting to put our foot down and ignore the subtleties of the issue.  We could simply say, "when a thing has focus it *must* get the focus ring indicator". However, since designs vary, what the particular focus ring should look like could vary too. On a white page background, a light blue outline might be really appropriate - but if your interface _is_ light blue, it might be necessary to consider this part of branding and perhaps provide something like:
```

Comment:
let's finish this pull then have a crack at it.  i'm open.

On Sat, Jul 25, 2015 at 1:29 PM, Alice <notifications@github.com> wrote:

> In docs/blog.md
> <https://github.com/bkardell/modality/pull/4#discussion_r35484925>:
>
> >
> > -Since keyboard users pretty much can't do anything without knowing where focus is, it's tempting to put our foot down and ignore the subtleties of the issue: we could simply say, "when a thing has focus it *must* get the focus ring indicator". Ok, since designs vary, what the particular focus ring should look like could vary too: on a white page background, a light blue outline might be really appropriate - but if your interface _is_ light blue, you might need to consider this in your branding and perhaps provide something like:
> > +Given this, it's tempting to put our foot down and ignore the subtleties of the issue.  We could simply say, "when a thing has focus it *must* get the focus ring indicator". However, since designs vary, what the particular focus ring should look like could vary too. On a white page background, a light blue outline might be really appropriate - but if your interface _is_ light blue, it might be necessary to consider this part of branding and perhaps provide something like:
>
> This paragraph still bothers me a bit (it's always bothered me but I
> haven't been able to come up with a solution) - the styling focus bit seems
> to sidetrack from our main point (even though I can see how it fits in with
> the next section).
>
> Perhaps the styling focus part needs to be a new paragraph, and finish up
> this one with a third sentence something like "However, this stance runs
> into problems before we even consider user context." Then the next
> paragraph begins "Since designs vary..."
>
> What do you think?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bkardell/modality/pull/4/files#r35484925>.
>



-- 
Brian Kardell :: @briankardell :: hitchjs.com